frontends/web/src/contexts/localization-provider.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react';
+import { useLocalizedPunctuation } from '../hooks/localized';
+import { AmountLocalizationContext } from './amount-localization-context';
+
+type TProps = {
+    children: ReactNode;
+  }
+
+export const AmountLocalizationProvider = ({ children }: TProps) => {
+  const { decimal, group } = useLocalizedPunctuation('');
```

Comment:
rightnow it is hardcoded to empty string, but this should `nativeLocale` 
```js
  const { nativeLocale } = useContext(AppContext);
```

Reply:
Oops. Sorry. Good catch.